### PR TITLE
CLOUDP-11772 (Mark) Add support for command line parsing

### PR DESCRIPTION
**Pull request by dgolub**
@markbenvenuto

### CODE REVIEW

File: cmdline.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2014 MongoDB, Inc.
```
Comment (markbenvenuto) [inline cmdline.go]: Update the year please

**Comment by markbenvenuto** [comment]
:+1:

File: cmdline.go (hunk 1 above)
Comment (dgolub) [inline cmdline.go]: Will do in a separate commit.

**Comment by dgolub** [comment]
Merged f705736.
